more_autograding_examples/cpp_updated_random_input_output/config/config.json

```diff
@@ -0,0 +1,15 @@
+{
+	"required_capabilities": "cpp",
+    "instructor_solution_compile_command": "g++ -std=c++11 -Wall -Wextra",
+    "testcases" : [
+
+        // *************** COMPILATION *****************
+        {
+            "type" : "Compilation",
+            "title" : "Compilation",
+            "command" : "clang++ -Wall -o a.out -- *.cpp",
+            "executable_name" : "a.out",
+            "points" : 3
+        }
+    ]
+}
```

Comment:
this example is incomplete, so I can't test it.  

Instead of making a whole new gradeable, as I said above, just replace the solution from the existing gradeable

Reply:
Got it.